### PR TITLE
excise all trace of approval-handler

### PR DESCRIPTION
**Pull request by spiffxp**
Thanks to @cjwagner's hard work, we've been able to roll out a prow-based replacement to all repos that used to use this

ref #3331

### CODE REVIEW

File: charts/mungegithub/chart/values.kubernetes.kubernetes.yaml [hunk 1]
```diff
@@ -12,11 +12,9 @@ config:
   http-cache-dir: /cache/httpcache
   organization: kubernetes
   project: kubernetes
-  # Make sure approval-handler and blunderbuss run before submit-queue.
+  # Make sure blunderbuss runs before submit-queue.
   # Otherwise it's going to take an extra-cycle to detect the label change.
-  # Run blunderbuss before approval-handler, so that we can suggest approvers
-  # based on assigned reviewer.
-  pr-mungers: blunderbuss,approval-handler,lgtm-after-commit,cherrypick-auto-approve,label-unapproved-picks,needs-rebase,path-label,size,stale-green-ci,block-path,release-note-label,comment-deleter,submit-queue,issue-creator,triage-filer,flakyjob-reporter,old-test-getter,docs-need-no-retest,sig-mention-handler
+  pr-mungers: blunderbuss,lgtm-after-commit,cherrypick-auto-approve,label-unapproved-picks,needs-rebase,path-label,size,stale-green-ci,block-path,release-note-label,comment-deleter,submit-queue,issue-creator,triage-filer,flakyjob-reporter,old-test-getter,docs-need-no-retest,sig-mention-handler
```
Comment (spiffxp) [inline charts/mungegithub/chart/values.kubernetes.kubernetes.yaml]: this is out of date relative to the actual configmap being used, this was pre misc-mungers/submit-queue split... maybe these should be removed? for now my goal was just to remove the approval-handler reference

File: charts/mungegithub/chart/values.kubernetes.contrib.yaml [hunk 2]
```diff
@@ -3,9 +3,7 @@ config:
   http-cache-dir: /cache/httpcache
   organization: kubernetes
   project: contrib
-  # Make sure approval-handler and blunderbuss run before submit-queue.
-  # Otherwise it's going to take an extra-cycle to detect the label change.
-  pr-mungers: approval-handler,blunderbuss,lgtm-after-commit,submit-queue,needs-rebase
+  pr-mungers: blunderbuss,lgtm-after-commit,submit-queue,needs-rebase
```
Comment (spiffxp) [inline charts/mungegithub/chart/values.kubernetes.contrib.yaml]: this is out of date relative to the actual configmap being used... maybe these should be removed? for now my goal was just to remove the approval-handler reference

**Comment by cjwagner** [comment]
Thanks for doing this Aaron! Its beautiful to see that many red lines in the mungegithub directory :grin: 
/lgtm

**Comment by kargakis** [comment]
@kubernetes/sig-testing-misc can we move mungegithub in **[kubernetes-graveyard](https://github.com/kubernetes-graveyard)** instead of merely deleting it from test-infra? There is still a use-case for using mungegithub over prow and that is folks with no operational experience of a k8s cluster AND they either use free tiers in public clouds where I expect they won't have access to create CRDs (but it's free to run a single monolithic deployment) or run in bare metal (no k8s/docker).

**Comment by BenTheElder** [comment]
I'm in favor of moving it to a graveyard, but we should definitely make it very clear that it's unmaintained and not really recommended / give a pointer to tide.

**Comment by kargakis** [comment]
SGTM

**Pull request merged**